vars/main.yml

```diff
@@ -230,10 +230,10 @@ manala_apt_keys_patterns:
     id:  86E50310
   sury_php_debian:
     url: https://packages.sury.org/php/apt.gpg
-    id: 4A7A714D
+    id:  4A7A714D
   sury_php_ubuntu:
     keyserver: hkp://keyserver.ubuntu.com:80
-    id: E5267A6C
+    id:        E5267A6C
```

Comment:
Yet I agree about aligning them all, I think it should be consistent all along the file (see e.g. https://github.com/manala/ansible-role-apt/pull/15/commits/f1f6e33b0ee9cfb3ff27937aa6f8f217cabdc914#diff-6d62b73b3f46cfa11f6466d08cf45303R215)